### ISSUE TITLE
Works great but application does not end (pause behavior)

### ISSUE THREAD

**Issue by ssy-lehmann** [libgdx/packr#202]
Hi,

I tried it and it works great. However after running the wrapped application the console hangs;
I've to press enter to end the applications. It behaves like a pause feature.

Did I miss someting?

**Comment by karlsabo** [comment]
Yeah, if you use `-c` on Windows it does that. No idea if there's a way to change the behavior. It's for debugging purposes so I never looked into it. PRs welcome :).
